Add Observability Operator page (#56)
rhobs/handbook: content/Products/ObservabilityOperator/README.md

# Observability Operator

[observability-operator](https://github.com/rhobs/observability-operator) is a Kubernetes operator which enables the management of Monitoring/Alerting stacks through Kubernetes CRDs. Eventually it might also cover Logging and Tracing.

The project relies heavily on the [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) library.

> More details to follow in this document

> For now visit observability-operator [README](https://github.com/rhobs/observability-operator#readme) page for details.

Reviewer: ```suggestion
Check the observability-operator's [README](https://github.com/rhobs/observability-operator#readme) page for more details.
```
